docs/source/crs/projections.rst

```diff
@@ -274,10 +274,19 @@ InterruptedGoodeHomolosine
     import matplotlib.pyplot as plt
     import cartopy.crs as ccrs
 
-    plt.figure(figsize=(6.9228, 3))
-    ax = plt.axes(projection=ccrs.InterruptedGoodeHomolosine())
-    ax.coastlines(resolution='110m')
-    ax.gridlines()
+    fig = plt.figure(figsize=(6.9228, 6))
+
+    ax1 = fig.add_subplot(2, 1, 1,
+                          projection=ccrs.InterruptedGoodeHomolosine(
+                              emphasis='land'))
+    ax1.coastlines(resolution='110m')
+    ax1.gridlines()
+
+    ax2 = fig.add_subplot(2, 1, 2,
+                          projection=ccrs.InterruptedGoodeHomolosine(
+                              central_longitude=-160, emphasis='ocean'))
+    ax2.coastlines(resolution='110m')
+    ax2.gridlines()
```

Comment:
This file is generated by `make_projection.py`; you need to update that first.

Reply:
Indeed, this code block is already in `make_projection.py`.

Reply:
Hmm, right, don't know why I didn't see it before.